crates/sui/tests/reconfiguration_tests.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
Let's rename this file to maybe reconfig_end_to_end_tests to distinguish from the other reconfiguration_tests